snowpack/src/config.ts

```diff
@@ -92,6 +93,7 @@ const configSchema = {
         fallback: {type: 'string'},
         bundle: {type: 'boolean'},
         open: {type: 'string'},
+        output: {type: 'string', enum: ['stream', 'dashboard']},
```

Comment:
I’m thinking about `output` as a CLI flag (e.g. `snowpack dev --output=stream`) and I’m slightly-against it because the way we’ve implemented these ([code](https://github.com/pikapkg/snowpack/blob/master/snowpack/src/config.ts#L177)), it seems like it could conflict with an `output` in another namespace (like `buildOptions.output`). 

Is there a more clear name that relates to logging than “output?” What about `logStyle`?

Reply:
eh tbh I think this is okay to wait until we hit that issue. We could just as easily choose a more specific name for that second "output" when we get to it. As mentioned above, the idea is to match the same "output" option that we support in the `run` plugin today.